incubator/hnc/e2e/tests/issues_test.go

```diff
@@ -0,0 +1,39 @@
+package test
+
+import (
+	"time"
+
+	. "github.com/onsi/ginkgo"
+)
+
+var _ = Describe("Issues", func() {
+
+	const (
+		nsParent = "issues-parent"
+		nsChild = "issues-child"
+	)
+
+	AfterEach(func() {
+		cleanupNamespaces(nsParent, nsChild)
+	}) 
+
+	It("Should test issue #328", func() {
+		// Setting up hierarchy with rolebinding that HNC doesn't have permission to copy.
+		mustRun("kubectl create ns", nsParent)
+		mustRun("kubectl create ns", nsChild)
+		mustRun("kubectl hns set", nsChild, "--parent", nsParent)
+		// cluster-admin is the highest-powered ClusterRole and HNC is missing some of
+		// its permissions, so it cannot propagate it.
+		mustRun("kubectl create rolebinding cluster-admin-rb -n", nsParent, 
+			"--clusterrole='cluster-admin' --serviceaccount=issue-parent:default")
+		// wait 2s
+		time.Sleep(2 * time.Second)
```

Comment:
This is no longer needed given that `runShouldContain` has an `Eventually` clause (which the bash script didn't). You should _never_ need an explicit sleep unless you're waiting to prove that something _hasn't_ happened.

Reply:
done